cmd/nodeconfig.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/celestiaorg/celestia-node/nodebuilder"
+	"github.com/celestiaorg/celestia-node/nodebuilder/node"
+	"github.com/celestiaorg/celestia-node/nodebuilder/p2p"
+)
+
+func Remove(nodeType node.Type, fsets ...*pflag.FlagSet) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "conf-remove",
+		Args:  cobra.NoArgs,
+		Short: "Remove current config",
+		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
+			return parseStorePath(cmd, nodeType)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			return nodebuilder.Remove(StorePath(ctx))
+		},
+	}
+	for _, set := range fsets {
+		cmd.Flags().AddFlagSet(set)
+	}
+
+	return cmd
+}
+
+func Reinit(nodeType node.Type, fsets ...*pflag.FlagSet) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "conf-reinit [config-path]",
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
why do you need to pass the path here explicitly? It should assume the path based on node type and whether `--p2p.network` flag was passed (or if a user specifies a `--node.store` directly, it should use that path)

Reply:
that is for the reinit file at the config path that contains the changes (either new sections or changes to existing sections to the config) that would be merged with the config present in node.store path